wiki/Beatmap/Pattern/Mania/Jacks/en.md

```diff
@@ -0,0 +1,31 @@
+---
+stub: true
+tags:
+  - jacks
+---
+
+# Jacks
+
+Within specific pattern context, **jacks** are typically used to refer to 3 or more consecutive notes within the same column. They are referred to as **jacks** since the motion required to play them mimics the motion of a jackhammer.
+
+![](img/jack.png "This image displays a jack within it's traditional usage of the word.")
+
+Commonly, the term **jack** can also refer to the general idea of having consecutive notes in the same column. Any of the following specifications can be referred to as just **jacks**.
+
+## Minijack
+
+A **minijack** is a type of **jack** which only features two notes and is the least demanding version of a **jack**.
```

Comment:
I would add that they are commonly snapped closer than regular jacks. As in, using 1/4 insead than 1/2 for example.

Reply:
Hmm, I think I wouldn't want to be too specific with that definition as your other suggestion (https://github.com/ppy/osu-wiki/pull/8595/files#r1065223090) already covers the idea that jacks are usually snapped at higher intervals. I think the mentioning of two notes only, which strays from the definition of jacks, should be enough to communicate the idea clearly